src/components/Categories/index.module.scss

```diff
@@ -0,0 +1,21 @@
+.root {
+  list-style-type: none;
+  display: flex;
+}
+
+
+.category {
+  color: var(--c-blue-1);
+  border: 1px solid currentColor;
+  padding: 0 12px;
+  border-radius: 10px;
+  margin-right: 12px;
+  height: 32px;
+  font-size: 12px;
+  display: flex;
+  align-items: center;
+
+  &:nth-child(2n) {
+    color: var(--c-purple-1);
+  }
+}
```

Comment:
Are we not using style lint to order CSS properties?

Reply:
ahaha didn't you remove that? I think I just don't have it configured on my editor. I'll take care of that 👍

Reply:
No, in the end I didn't remove anything ..

Reply:
👍